pkg/handlers/generic/lifecycle/csi/handler.go

```diff
@@ -27,7 +27,7 @@ type CSIProvider interface {
 	Apply(
 		context.Context,
 		v1alpha1.CSIProvider,
-		*v1alpha1.DefaultStorage,
+		v1alpha1.DefaultStorage,
```

Comment:
Why not a pointer anymore?

Reply:
This change makes DefaultStorage a required field. It can't be nil, and so there's no need to represent that possibility.